File: src/edu/gbcg/DBSelector/RedditComments/CommentSetMapper.java
package edu.gbcg.DBSelector.RedditComments;

import edu.gbcg.DBSelector.RSMapper;
import edu.gbcg.dbcreator.Reddit.Comments;

import java.sql.ResultSet;
import java.util.List;
import java.util.Map;

public class CommentSetMapper extends RSMapper {
    public CommentSetMapper(Map<String, String> map){ super(map); }
    public CommentSetMapper(){ super(); }
    public List<RSMapper> buildMappers(ResultSet rs){
        return buildMappers_impl(rs, Comments.getColumnsForDB());
    }
}

File: src/main/kotlin/edu/antevorta/dbInteraction/dbSelector/reddit/comments/CommentSetMapper.kt
/*
 * Copyright (c) 2018 Sean Grimes. All Rights Reserved.
 * License: MIT
 */

package edu.antevorta.dbInteraction.dbSelector.reddit.comments

import edu.antevorta.dbInteraction.columnsAndKeys.RedditComs
import edu.antevorta.dbInteraction.dbSelector.RSMapper
import org.json.JSONObject
import java.sql.ResultSet

class CommentSetMapper: RSMapper {
    constructor(map: Map<String, String>): super(map)
    constructor(jsonObject: JSONObject): super(jsonObject)
    constructor(): super()
    override fun buildMappers(rs: ResultSet): MutableList<RSMapper> {
        return buildMappersImpl(rs, RedditComs.columnNames())
    }
}
